playbooks/roles/learner-profile-dashboard/defaults/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+LPD_USER_NAME: "lpd"
+LPD_USER_HOME: "/home/{{ LPD_USER_NAME }}"
+LPD_APP_LOCATION: "{{ LPD_USER_HOME }}/repository"
+LPD_VENV: "{{ LPD_USER_HOME }}/venv"
+LPD_PYTHON: "{{ LPD_VENV }}/bin/python"
+LPD_MANAGE_PY: "{{ LPD_PYTHON }} {{ LPD_APP_LOCATION }}/manage.py "
+
+LPD_LOG_DOWNLOAD_USER: logs-download
+LPD_LOG_DOWNLOAD_AUTHORIZED_KEYS: []
+LPD_LOG_DOWNLOAD_USER_HOME: "/home/{{ LPD_LOG_DOWNLOAD_USER }}"
+LPD_LOG_DOWNLOAD_LOG_DIR: "{{ LPD_LOG_DOWNLOAD_USER_HOME }}/logs"
+LPD_LOG_DOWNLOAD_DB_DIR: "{{ LPD_LOG_DOWNLOAD_USER_HOME }}/db"
+LPD_LOG_TARSNAP_ARCHIVE: lpd-logs
+
+LPD_LOG_DOWNLOAD_VOLUME_DEVICE_ID: /dev/vdc
+LPD_LOG_DOWNLOAD_VOLUME_FILESYSTEM: ext4
+
+# Note: when overriding these leave DJANGO_SETTINGS_MODULE inside
```

Comment:
@tomaszgy You might have copied this from somewhere else (?), but I'm not sure what `leave DJANGO_SETTINGS_MODULE inside` means?

Reply:
@itsjeyd yeah, I reused dalite ansible role to create that. This variable [is used by Gunicorn](http://docs.gunicorn.org/en/stable/run.html#django) and I guess that what the author of the comment meant was that this variable shouldn't be removed out of `LPD_ENV` (as `LPD_ENV` is later on passed to Gunicorn). Shall we reword it to `leave DJANGO_SETTINGS_MODULE inside LPD_NV`?

Reply:
@itsjeyd I went ahead and reworded it to that but will revert it if it doesn't make sense - please let me know :)

Reply:
@tomaszgy That makes sense, thanks :)